src/main/java/frc/robot/commands/ToggleFieldAbsoluteCommand.java

```diff
@@ -22,12 +22,12 @@ public ToggleFieldAbsoluteCommand(ShuffleboardDriverControls  prefs) {
 
   @Override
   public void execute() {
-	driverPrefs.toggleFieldAbsolute();
+	  driverControls.toggleFieldAbsolute();
   }
 
   @Override
   public boolean runsWhenDisabled() {
```

Comment:
runsWhenDisabled() needs to return true so that the drive mode can be toggled when the robot is disabled before the match.

Reply:
woo good catch! yep!